src/broker/broker_cas_cci.h

```diff
@@ -192,7 +192,8 @@ typedef enum
   CCI_SCH_IMPORTED_KEYS,
   CCI_SCH_EXPORTED_KEYS,
   CCI_SCH_CROSS_REFERENCE,
-  CCI_SCH_LAST = CCI_SCH_CROSS_REFERENCE
+  CCI_SCH_ATTR_INCLUDE_SYNONYM,
+  CCI_SCH_LAST = CCI_SCH_ATTR_INCLUDE_SYNONYM
```

Comment:
You should also reflect this modification to **cas_cci.h** of cubrid-cci repo.

Reply:
All right.
I will PR the cubrid-cci repo.